cc/dual_net/batching_dual_net.cc

```diff
@@ -17,15 +17,19 @@
 #include <utility>
 
 #include "absl/memory/memory.h"
+#include "absl/time/clock.h"
 #include "cc/logging.h"
+#include "wtf/macros.h"
 
 namespace minigo {
 
 namespace internal {
 
 ModelBatcher::ModelBatcher(std::unique_ptr<DualNet> model_impl,
-                           size_t buffering)
-    : model_impl_(std::move(model_impl)), buffering_(buffering) {}
+                           size_t buffer_count)
+    : model_impl_(std::move(model_impl)),
+      buffer_count_(buffer_count),
```

Comment:
Can you add a note about what buffer_count is used for (and how reasonable values are 0-4?)

Reply:
added a comment to the header